hazelcast-ra/hazelcast-jca/src/main/java/com/hazelcast/jca/ManagedConnectionImpl.java

```diff
@@ -55,16 +55,16 @@
     private HazelcastTransactionImpl localTransaction;
 
     public ManagedConnectionImpl(ConnectionRequestInfo cxRequestInfo, ManagedConnectionFactoryImpl factory) {
-        this.setLogWriter(factory.getLogWriter());
-        log(Level.FINEST, "ManagedConnectionImpl");
+        this.id = ID_GEN.incrementAndGet();
```

Comment:
This file contains the biggest changes, since I re-ordered the field assignments according to the field order in the class and moved the logging part to the end of the constructor.